tests/test_android.py

```diff
@@ -0,0 +1,82 @@
+#!/usr/bin/env python3
+#
+# Cross Platform and Multi Architecture Advanced Binary Emulation Framework
+# Built on top of Unicorn emulator (www.unicorn-engine.org)
+
+
+import unittest
+import os
+import sys
+sys.path.append("..")
+
+from qiling.os.utils import ql_definesyscall_return
+from qiling import *
+from qiling.os.posix.syscall.unistd import ql_syscall_pread64
+
+# syscalls that need to be implemented for android
+def syscall_getrandom(ql, buf, buflen, flags, null0, null1, null2):
+    data = None
+    regreturn = None
+    try:
+        data = os.urandom(buflen)
+        ql.uc.mem_write(buf, data)
+        regreturn = len(data)
+    except:
+        regreturn = -1
+
+    ql.nprint("getrandom(0x%x, 0x%x, 0x%x) = %d" %
+              (buf, buflen, flags, regreturn))
+
+    if data:
+        ql.dprint(0, "[+] getrandom() CONTENT:")
+        ql.dprint(0, str(data))
+    ql_definesyscall_return(ql, regreturn)
```

Comment:
hi, we did a change so you can call
```
ql.os.definesyscall_return
```

can you update and try it. thanks

Reply:
Hi, I'm not seeing that change in either dev or master branches yet (I'm getting "'QlOsLinux' object has no attribute 'definesyscall_return'" when I run). Has it been merged in yet?

Reply:
Haha, yes it has, I was just failing to see it, my bad. Changed over to ql.os.definesyscall_return.